### PR TITLE
Remove the 16px message icons from the misc directory.

### PR DESCRIPTION
**Pull request by jenlampton**
Fixes https://github.com/backdrop/backdrop-issues/issues/3437

This PR also adds our new `info` color to info type messages.

### CODE REVIEW

File: core/modules/dblog/css/dblog.css [hunk 1]
```diff
@@ -68,18 +68,22 @@
...
 }
 .admin-dblog .dblog-error .icon:before,
 .admin-dblog .dblog-critical .icon:before,
 .admin-dblog .dblog-alert .icon:before,
 .admin-dblog .dblog-emergency .icon:before {
-  background: #EE3D23 url(../../../misc/message-16-error.png) no-repeat center center;
-}
-.admin-dblog .dblog-debug .icon:before,
-.admin-dblog .dblog-info .icon:before,
-.admin-dblog .dblog-notice .icon:before {
```
Comment (opi) [inline core/modules/dblog/css/dblog.css:83]: we loose notice ?

**Comment by jenlampton** [comment]
The default is to use the notice icon now.

File: core/modules/dblog/css/dblog.css (hunk 1 above)
Comment (quicksketch) [reply core/modules/dblog/css/dblog.css:83]: Looks like it...

Comment (jenlampton) [reply core/modules/dblog/css/dblog.css:83]: No, the default is notice now, instead. See line 71.

Comment (jenlampton) [reply core/modules/dblog/css/dblog.css:83]: > we loose notice ?

Nope, see line 71.

**Comment by quicksketch** [comment]
Thanks! As this isn't a bug fix I've only merged into 1.x, and not the current minor release.

**Pull request merged**